Keas.Mvc/Services/ReportService.cs

```diff
@@ -291,5 +293,26 @@ public async Task<IList<KeyReportModel>> Keys(Team team, string teamSlug, bool i
 
             return keys;
         }
+
+        public async Task<IList<IncompleteDocumentReportModel>> IncompleteDocuments(Team team, string teamSlug)
+        {
+            if (team == null)
+            {
+                team = await _context.Teams.SingleAsync(a => a.Slug == teamSlug);
+            }
+
+            var people = await _context.People.AsNoTracking().Include(a => a.Documents)
+                .Where(a => a.TeamId == team.Id && a.Documents.Any(x => x.Active && x.Status != "Completed"))
+                .Select(s => new IncompleteDocumentReportModel()
+                {
+                    Name = s.Name,
+                    Email = s.Email,
+                    PersonId = s.Id,
+                    IncompleteDocumentCount = s.Documents.Count(w => w.Active && w.Status != "Completed"),

```

Comment:
@jSylvestre how about we change to status == "sent".  Since there are other completions like Cancelled, Voided, Denied, etc that are possible end states

Reply:
@srkirkland So, if it is anything other than sent or completed, will someone go in and set it to inactive? (Delete it).
My thought is the admin want to know about all that are not completed. And if there is some other status, they should "fix" it.

Reply:
Those statuses all come from docusign.  An example would be if the document isn't signed within 120 days it'll go to some Void status or similar.  There is nothing that can be done to fix it, since that is the correct final state.  I also think there might be some way for the signer to reject signing the document, which is a similar case.

Reply:
@srkirkland  If a "delete document" action is added to the documents container, then I think this will work well.
We would want to report all people who have not signed a document. This would include the Void and rejected status.
In these cases, the document admin should act on these cases. Probably by deleting the document, then adding it again. Unless it was done in error.

Reply:
Yea I think that sounds good, though I'm in no rush to add the document delete functionality to my nice clean table ;0